hypershift-operator/controllers/hostedcluster/internal/platform/openstack/openstack.go

```diff
@@ -66,9 +69,66 @@ func (a OpenStack) ReconcileCAPIInfraCR(ctx context.Context, client client.Clien
 	}); err != nil {
 		return nil, err
 	}
+
+	err := reconcileOpenStackImageCR(ctx, client, createOrUpdate, hcluster, controlPlaneNamespace)
+	if err != nil {
+		return nil, err
+	}
+
 	return openStackCluster, nil
 }
 
+func reconcileOpenStackImageCR(ctx context.Context, client client.Client, createOrUpdate upsert.CreateOrUpdateFN, hcluster *hyperv1.HostedCluster, controlPlaneNamespace string) error {
+	openStackImage := orc.Image{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhcos" + hcluster.Name,
```

Comment:
```suggestion
			Name:      "rhcos-" + hcluster.Name,
```

Reply:
We may also want to name it by the release image version so it can be shared between clusters 🤔

Reply:
I don't know if the resource could be shared across multiple clusters, that's an interesting idea. I'll investigate for sure.

Reply:
I thought about it today and if a customer want a shared image, they can just pre-create it and use `--openstack-node-image-name`.